test/RunTests

```diff
@@ -2323,4 +2323,13 @@ train-sets/ref/slates_simple_w_interactions.stderr
 {VW} -d train-sets/squarecb_load.dat --cb_explore_adf -q UA --squarecb -i models/sqcb_ld.model -t
     train-sets/ref/squarecb_load.stderr
 
+# Test 316: bfgs preconditioner training
+{VW} -c -k -d train-sets/rcv1_small.dat --bfgs --passes 12 -f models/bfgs_precon.model --save_resume
+    train-sets/ref/bfgs_train.stderr
+    
+# Test 317: bfgs preconditioner testing
```

Comment:
What is the purpose of the second test? If the fixed overflowing check is triggered by this dataset, it probably should be already captured by the output from the first test